diff --git a/node_modules/postcss-normalize/index.mjs b/node_modules/postcss-normalize/index.mjs
index 03594d3..7c2f333 100644
--- a/node_modules/postcss-normalize/index.mjs
+++ b/node_modules/postcss-normalize/index.mjs
@@ -1,5 +1,5 @@
 import postcssBrowserComments from 'postcss-browser-comments';
-import Module from 'module';
+import { createRequire} from 'node:module'
 import path from 'path';
 import { URL } from 'url';
 import fs from 'fs';
@@ -8,9 +8,8 @@ import postcss from 'postcss';
 const assign = (...objects) => Object.assign(...objects);
 const create = (...objects) => assign(Object.create(null), ...objects);

-const currentURL = import.meta.url;
-const currentFilename = new URL(currentURL).pathname;
-const currentDirname = path.dirname(currentFilename); // get resolved filenames for normalize.css
+// create package path resolver in an esm-compatible fashion
+const {resolve: requireResolve} = createRequire(import.meta.url)

 const normalizeCSS = resolve('@csstools/normalize.css');
 const normalizeDir = path.dirname(normalizeCSS);
@@ -54,11 +53,7 @@ const resolvedFilenamesById = create({
 }); // get the resolved filename of a package/module

 function resolve(id) {
-  return resolve[id] = resolve[id] || Module._resolveFilename(id, {
-    id: currentFilename,
-    filename: currentFilename,
-    paths: Module._nodeModulePaths(currentDirname)
-  });
+  return resolve[id] = resolve[id] || requireResolve(id)
 }

 const cache$1 = create();
