diff --git a/KeepAliveProvider.tsx b/KeepAliveProvider.tsx
index 7dba1adde3ea1b66f98dbb99566f725df6d4cd6c..c15a81fe3639f8d49fcf03d28b302ec78a63d86d 100644
--- a/KeepAliveProvider.tsx
+++ b/KeepAliveProvider.tsx
@@ -1,4 +1,4 @@
-import CacheContext from "./CacheContext";
+import CacheContext from "./cacheContext";
 import cacheReducer from "./cacheReducer";
 import CacheType from "./cache-type";
 import { CacheState, MountFunction, ScrollFunction } from "./global";
diff --git a/withKeepAlive.tsx b/withKeepAlive.tsx
index 1f4d9cb835be9562c194d7a8789a103c7a9cfa5f..db1e315c42620663ef6d583a1cc505e89f1c9f39 100644
--- a/withKeepAlive.tsx
+++ b/withKeepAlive.tsx
@@ -1,4 +1,4 @@
-import CacheContext from "./CacheContext";
+import CacheContext from "./cacheContext";
 import CacheType from "./cache-type";
 import { useCallback, useContext, useEffect, useRef } from "react";
 import { WithKeepAlive } from "./global";
