diff --git a/node_modules/react-cosmos/dist/dom/index.js b/node_modules/react-cosmos/dist/dom/index.js
index 5828849..ed64872 100644
--- a/node_modules/react-cosmos/dist/dom/index.js
+++ b/node_modules/react-cosmos/dist/dom/index.js
@@ -19,7 +19,7 @@ Object.defineProperty(exports, "rendererId", {
 
 var _react = _interopRequireDefault(require("react"));
 
-var _reactDom = require("react-dom");
+var _reactDomClient = require("react-dom/client");
 
 var _container = require("./container");
 
@@ -39,9 +39,10 @@ function mountDomRenderer(_ref) {
       decorators = _ref.decorators,
       onErrorReset = _ref.onErrorReset;
   var domContainer = (0, _container.getDomContainer)(rendererConfig.containerQuerySelector);
-  (0, _reactDom.render)( /*#__PURE__*/_react.default.createElement(_DomFixtureLoader.DomFixtureLoader, {
+  var root = (0, _reactDomClient.createRoot)(domContainer);
+  root.render( /*#__PURE__*/_react.default.createElement(_DomFixtureLoader.DomFixtureLoader, {
     fixtures: fixtures,
     decorators: decorators,
     onErrorReset: onErrorReset
-  }), domContainer);
+  }));
 }
\ No newline at end of file
diff --git a/node_modules/react-cosmos/dist/generateWebExport.js b/node_modules/react-cosmos/dist/generateWebExport.js
index a6421d2..672e176 100644
--- a/node_modules/react-cosmos/dist/generateWebExport.js
+++ b/node_modules/react-cosmos/dist/generateWebExport.js
@@ -24,7 +24,9 @@ function _generateWebExport() {
         switch (_context.prev = _context.next) {
           case 0:
             _context.next = 2;
-            return (0, _generateExport.generateExport)([_webpack.webpackExport]);
+
+            // As we have vite bundler and do not need webpack to be run
+            return (0, _generateExport.generateExport)([]);
 
           case 2:
           case "end":
