diff --git a/node_modules/vis-timeline/peer/esm/vis-timeline-graph2d.js b/node_modules/vis-timeline/peer/esm/vis-timeline-graph2d.js
index 35bbc84..7259908 100644
--- a/node_modules/vis-timeline/peer/esm/vis-timeline-graph2d.js
+++ b/node_modules/vis-timeline/peer/esm/vis-timeline-graph2d.js
@@ -27768,7 +27768,7 @@ var ClusterGenerator = /*#__PURE__*/function () {
         toTime: this.itemSet.body.util.toTime
       };
       var title = titleTemplate.replace(/{count}/, clusterItems.length);
-      var clusterContent = '<div title="' + title + '">' + clusterItems.length + '</div>';
+      var clusterContent = options.contentTemplate ? options.contentTemplate(clusterItems) : '<div title="' + title + '">' + clusterItems.length + '</div>';
 
       var clusterOptions = _Object$assign({}, options, this.itemSet.options);
 
@@ -31709,6 +31709,10 @@ var allOptions$1 = {
       'boolean': bool$1,
       'undefined': 'undefined'
     },
+    contentTemplate: {
+      'function': 'function',
+      'undefined': 'undefined'
+    },
     __type__: {
       'boolean': bool$1,
       object: object$1
